pipeline/frontend/yaml/types/base/map.go

```
@@ -21,13 +21,13 @@ import (
 )
 
 // SliceOrMap represents a map of strings, string slice are converted into a map
-type SliceOrMap map[string]string
+type SliceOrMap map[string]any
```

is slice or map only used at environment? else I dont want to loosen other datatypes unrelated to this change

---

If we dont throw an error out already ... we should not silently drop it within one version

First depricate 
next error
then remove

---

yes but Labels are affected ... anyway I guess it's fine as we did deprecated them a long time ago

---

e.g. we have to copy the `SliceOrMap` type to ~ `SliceOrMapAny` and still use the old SliceOrMap for Labels